setup.py

```diff
@@ -291,10 +309,20 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform_darwin', [platform_darwin_source]))
 
+        
+        
+def parse(root, describe_command=None):
+    return subprocess.check_output("git describe --tags --long").decode().strip()
```

Comment:
hmm, does this mean a user (not developer) would need git installed to install borg?